pkg/scheduler/apis/config/v1beta2/defaults.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Also confirm if this file is up to date.

And it's 2021 now :)

Reply:
This is up to date

Reply:
Can you change the comments to 2021 please?